packages/gatsby-source-mongodb/src/gatsby-node.js

```diff
@@ -1,6 +1,8 @@
 const MongoClient = require(`mongodb`).MongoClient
 const crypto = require(`crypto`)
 const prepareMappingChildNode = require(`./mapping`)
+const sanitizeName = require(`./sanitize-name`)
+const _ = require(`lodash`)
```

Comment:
Is this used? If not:

```suggestion
```

Reply:
It is used on line 76 and 77

Reply:
Neither of those lines are using lodash?

https://github.com/gatsbyjs/gatsby/blob/f2e3a89691db8036963a33852039ffeb0289a4ac/packages/gatsby-source-mongodb/src/gatsby-node.js#L76-L77

Reply:
Yes, they are not using lodash. Silly mistake, sorry